### PR TITLE
Update Agent Flow Linux install topic

### PR DESCRIPTION
**Pull request by clayton-cornell**
#### PR Description

Update the Linux install steps for Grafana Agent Flow

* Minor layout changes to align with Static docs where applicable.
* Add in missing SUSE/openSUSE installation steps.

#### PR Checklist

- [ ] CHANGELOG updated
- [ ] Documentation added
- [ ] Tests updated

### CODE REVIEW

File: docs/sources/flow/install/linux.md [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
-title: Linux systems
+title: Linux
 weight: 300
 ---
 
 # Install Grafana Agent Flow on Linux systems
 
-Grafana Agent Flow can be installed as a systemd service on various AMD64 and
-ARM64 Linux systems:
+You can install Grafana Agent Flow as a systemd service on Linux
```
Comment (clayton-cornell) [inline docs/sources/flow/install/linux.md:8]:
```suggestion
You can install Grafana Agent Flow as a systemd service on Linux.
```

File: docs/sources/flow/install/linux.md [hunk 2]
```diff
@@ -117,7 +125,7 @@ To change the configuration file used by the service, perform the following step
 1. Edit the environment file for the service:
 
    * Debian-based systems: edit `/etc/default/grafana-agent-flow`
-   * RedHat-based systems: edit `/etc/sysconfig/grafana-agent-flow`
+   * RedHat ro SUSE-based systems: edit `/etc/sysconfig/grafana-agent-flow`
```
Comment (clayton-cornell) [inline docs/sources/flow/install/linux.md:128]:
```suggestion
   * RedHat or SUSE-based systems: edit `/etc/sysconfig/grafana-agent-flow`
```

**Review by jcreixell — APPROVED** [review]
LGTM

File: docs/sources/flow/install/linux.md [hunk 3]
```diff
@@ -55,54 +49,68 @@ or RedHat Enterprise Linux), complete the following steps:
...
+   wget -q -O gpg.key https://apt.grafana.com/gpg.key
+   sudo rpm --import gpg.key
+   sudo zypper addrepo https://rpm.grafana.com grafana
+   ```
+
+1. Update the repository and install Grafana Agent:
+
+   ```shell
+   sudo zypper update
+   sudo zypper install grafana-agent
```
Comment (rfratto) [inline docs/sources/flow/install/linux.md:80]: @clayton-cornell Should this be grafana-agent-flow?

Comment (clayton-cornell) [inline docs/sources/flow/install/linux.md:80]:
```suggestion
   sudo zypper install grafana-agent-flow
```

Comment (clayton-cornell) [reply docs/sources/flow/install/linux.md]: @rfratto D'oh! Yes it should. Fixed.

**Pull request merged**